admin.py

```diff
@@ -374,6 +370,7 @@ class DonationForm(djforms.ModelForm):
   event = make_admin_ajax_field(tracker.models.Donation, 'event', 'event', initial=latest_event_id)
   class Meta:
     model = tracker.models.Donation
+    exclude = ('', '')
```

Comment:
Might be, but I probably just put in the first thing that popped into my head and copy-pasted it all over.